about.html

```diff
@@ -0,0 +1,135 @@
+
+      <div class="content">
+        <h1>"Welcome! to the Marketing world"</h1>
+        <h2>A GLOBAL GATHERING <br>OF BRAND BUILDERS & MARKETERS</h2>
+        <div class="inner-content">
+          <p>The Marketing Conference spotlights the latest innovations and breakthroughs in the marketing and advertising industry. Connect with the global community and join thought leaders and storytellers as they explore visionary ideas that engage and
+            inspire
+          </p>
+        </div>
+        <div class="inner-content-bottom">
+          <p>Please contact as per Email for any further questions about global Gathering</p>
+          <h3>globalgather@ggather.org</h3>
+          <div class="under-line"> </div>
+
+        </div>
+      </div>
+
+    </div>
+    <div class="featured-speakers">
+      <div>
+        <h2>About Marketing Conference That’s Bringing This Event To Life!</h2>
+        <div class="under-line"> </div>
+      </div>
+      <div>
+        <p>Marketing Conference is a premier marketing and advertising conference that will draw eminent leaders of the community from all over the world. <br><span class="content-hide">it aspires to help businesses connect with their customers better, enhance their visibility, and build their brand by learning from the best marketers. <br> The conference will unveil the latest technologies, research, and ideas that sell in the realms of content creation, branding, marketing, and advertising.industry. </span></p>
+      </div>
+      <div class="about-content-logo"><img src="./images/new-logo.png" alt="about logo" /></div>
+
+
+    </div>
+    <hr>
+    <div class="featured-speakers">
+      <div>
+        <h2>See the coming Global Marketing Conferences</h2>
+        <div class="under-line"> </div>
+      </div>
+      <div>
+        <p>Take a look at the given two global Marketing Conferences which are taking place in USA</p>
+      </div>
+      <div class="summits">
+
+        <div class="summit-img"><img src=".images/img_banner_agenda_d.png" alt="summit 2022"></div>
+        <div class="over-blur">USA 17 Date: 18th - 19th March, 2022 Venue:Festival City, USA</div>
+        <div class="summit-img"><img src="./images/img_banner_agenda_u.png" alt="summit 2022"></div>
+
+
+
+
+      </div>
+    </div>
+  </section>
+
+
+
+  <section class="footer">
+    <div class="footer-bottom-section-about">
+      <div class="footer-bottom">
+        <div class="footer-bottom-logo"><img src="./images/footer-new-black.png" alt="bottom logo"></div>
+        <div class="footer-bottom-content">
+          <h4>© 2022 Global Marketing Conference. All Rights Reserved.</h4>
+          <p>we would like to assure you that we are taking all essential COVID-19 protocols and precautions <br> in accordance with the regulations set by the state authorities.
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
```

Comment:
Kindly try to match the `Behance` design for the footer section in the mobile version on the about page, The footer is split into two parts, and Please change the `section` tag with the `footer` tag for semantic purposes
| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/157549216-eccced1d-c140-4fb1-9f6e-8aad7bace5e1.png) | ![image](https://user-images.githubusercontent.com/24703171/157549351-f88008bc-3976-4731-a487-87a58069afdf.png) |